timbermill-local/src/test/java/com/datorama/oss/timbermill/ElasticsearchClientTest.java

```diff
@@ -0,0 +1,97 @@
+package com.datorama.oss.timbermill;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.UUID;
+
+import org.elasticsearch.action.bulk.BulkItemResponse;
+import org.elasticsearch.action.bulk.BulkRequest;
+import org.elasticsearch.action.bulk.BulkResponse;
+import org.elasticsearch.action.update.UpdateRequest;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.script.Script;
+import org.elasticsearch.script.ScriptType;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.datorama.oss.timbermill.common.Constants;
+import com.datorama.oss.timbermill.common.DbBulkRequest;
+import com.datorama.oss.timbermill.common.SQLJetDiskHandler;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ElasticsearchClientTest {
```

Comment:
Change these tests accordingly to the changes in ElasticsearchClient, I don't think `handleBulkRequestFailure` should change the request itself